ui/perfherder/graphs/TableView.jsx

```diff
@@ -28,169 +84,177 @@ const TableView = ({
                   rel="noopener noreferrer"
                 >
-                  compare
+                  {revision.slice(0, 13)}
                 </a>
-              </div>
+              )}
             </div>
           );
         },
+        headerClassName:
+          'text-wrap d-flex justify-content-center align-items-center',
         getProps: (state, rowInfo) => setHighlightedRow(rowInfo),
-      });
+      },
+    ];
 
-      // populate array with only data points
-      item.data.forEach((dataPoint, dataIndex) => {
-        const prevFlotDataPointIndex = dataIndex - 1;
-
-        // calculate delta values
-        const valueY = dataPoint.y;
-        const v0 =
-          prevFlotDataPointIndex !== -1
-            ? item.data[prevFlotDataPointIndex].y
-            : valueY;
-        const deltaValue = valueY - v0;
-        const deltaPercent = valueY / v0 - 1;
-
-        // get push link
-        let prevRevision;
-        let pushUrl;
-        if (prevFlotDataPointIndex !== -1) {
-          prevRevision = item.data[prevFlotDataPointIndex].revision;
-          const repoModel = new RepositoryModel(
-            projects.find(
-              repositoryName => repositoryName.name === item.repository_name,
-            ),
-          );
-          pushUrl = repoModel.getPushLogRangeHref({
-            fromchange: prevRevision,
-            tochange: dataPoint.revision,
-          });
-        }
+    testData.forEach((item, index) => {
+      if (item.visible) {
+        const dataKey = `data${index}`;
 
-        const jobsUrl = getJobsUrl({
-          repo: item.repository_name,
-          revision: dataPoint.revision,
-          selectedJob: dataPoint.jobId,
-          group_state: 'expanded',
-        });
+        // add columns for each test
+        columns.push(setTestColumnHeader(item, dataKey, setHighlightedRow));
 
-        const compareUrl = `#/comparesubtest${createQueryParams({
-          originalProject: item.repository_name,
-          newProject: item.repository_name,
-          originalRevision: prevRevision,
-          newRevision: dataPoint.revision,
-          originalSignature: item.parentSignature || item.signature_id,
-          newSignature: item.parentSignature || item.signature_id,
-          framework: item.framework_id,
-        })}`;
-
-        allDataPoints.push({
-          [id]: item.name,
-          date: moment(dataPoint.x).format('MMM DD, h:mm:ss a'),
-          [value]: numeral(dataPoint.y).format('0,0.[00]'),
-          highlighted:
-            highlightAlerts &&
-            highlightedRevisions.some(
-              item => item && dataPoint.revision.includes(item),
+        // populate array with only data points
+        item.data.forEach((dataPoint, dataIndex) => {
+          allDataPoints.push(
+            getRowDataPoint(
+              item,
+              dataIndex,
+              dataPoint,
+              highlightAlerts,
+              highlightedRevisions,
```

Comment:
What changed in this file from the last commit?

Reply:
Some functions were created and I changed the data structured sent to the table. I basically removed [this](https://github.com/mozilla/treeherder/pull/6001/commits/6183d6c3115cb090d203639a60101a071715ad4d#diff-fce1c2201d8b61dea9a7b9418006ace2R60-R63) and moved it into an object. I thought it was cleaner.

[Before](https://github.com/mozilla/treeherder/pull/6001/commits/6183d6c3115cb090d203639a60101a071715ad4d#diff-fce1c2201d8b61dea9a7b9418006ace2R160-R174) | [Current](https://github.com/mozilla/treeherder/pull/6001/files#diff-fce1c2201d8b61dea9a7b9418006ace2R161-R177)

(there are some lines highlighted, but apparently Github doesn't jump to them when you click the link)